docs/admin/architectures/2k-users.md

```diff
@@ -0,0 +1,59 @@
+# Reference Architecture: up to 2,000 users
+
+In the 2,000 users architecture, there is a moderate increase in traffic,
+suggesting a growing user base or expanding operations. This setup is
+well-suited for mid-sized companies experiencing growth or for universities
+seeking to accommodate their expanding user populations.
+
+Users can be evenly distributed between 2 regions or be attached to different
+clusters.
+
+**Target load**: API: up to 300 RPS
+
+**High Availability**: The mode is _disabled_, but administrators may consider
+enabling it for deployment reliability.
+
+## Hardware recommendations
+
+### Coderd nodes
+
+| Users       | Node capacity        | Replicas | GCP             | AWS         | Azure             |
+| ----------- | -------------------- | -------- | --------------- | ----------- | ----------------- |
+| Up to 2,000 | 4 vCPU, 16 GB memory | 2        | `n1-standard-4` | `t3.xlarge` | `Standard_D4s_v3` |
```

Comment:
Perhaps we can clarify here a bit. A reader might wonder if we mean 4 vCPU, 16 GB / replica or if we mean put both replicas on the single 4 vCPU node.

I wonder if an additional column would be beneficial? Something like:

```markdown
| ... | Nodes | Replicas/node | ...
|     | 2     | 1             | ...
```

Alternatively, we could follow the syntax used below: `2 nodes / 1 coderd each`.

PS. This memory recommendation is a bit on the higher end, but I think it's a good choice because it leaves room for increased web terminal/app traffic.

Reply:
Looking at our past scaling results, we have already added more features (= more CPU usage), so leaving extra capacity may be a good decision :)